src/CodeGen_C.cpp

```diff
@@ -551,6 +551,20 @@ void CodeGen_C::compile(const Module &input) {
     }
 
     if (!is_header()) {
+        // Emit any external-code blobs that are C++.
+        for (const ExternalCode &code_blob : input.external_code()) {
+            if (code_blob.is_c_plus_plus_source()) {
+                stream << "\n";
+                stream << "// Begin External Code: " << code_blob.name() << "\n";
+                for (auto c : code_blob.contents()) {
```

Comment:
I think

    stream.write(code_blob.contents().data(), code_blob.contents().size());

would be better.